### PR TITLE
lapindrome checker in cpp

### PR DESCRIPTION
**Pull request by tbhaxor**
**Fixes issue:** #1677

**Changes:**
Added lapindrome checker in cpp

### CODE REVIEW

File: code/string_algorithms/lapindrome_checker/lapindrome_checker.cpp [hunk 1]
```diff
@@ -0,0 +1,36 @@
+// part of cosmos from opengenus foundation
+
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+bool is_lapindrome(string _str_)
+{
+	int len = _str_.length();
```
Comment (arnavb) [inline code/string_algorithms/lapindrome_checker/lapindrome_checker.cpp]: Useless variable.

File: code/string_algorithms/lapindrome_checker/lapindrome_checker.cpp [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	sort(first.begin(), first.end());
+	sort(second.begin(), second.end());
+
+	return first == second;
+
+}
+
+int main()
+{
+	if(is_lapindrome("abccba"))
```
Comment (arnavb) [inline code/string_algorithms/lapindrome_checker/lapindrome_checker.cpp]: Might want to generalize this, and add actual tests with Catch2.

**Comment by tbhaxor** [comment]
@arnavb  have u checked ur mail
i hav sent u some issues regarding using git on windows

**Comment by tbhaxor** [comment]
@AdiChat i request you to please delete `code/string_algorithm/lapindrome_checker`

*Why i am asking you to delete ?*
+ This was added by me, due to some issue my contrib is not shown
+ Some issues are notified by @arnavb but i can't modify them (i have mailed my issue regarding git on @arnavb email)
+ I am adding new code in `cpp` and `python`

**Pull request closed without merging**